spell/spell-reviewer-goerli-checklist.md

```diff
@@ -9,11 +9,12 @@ Spell Actions:
 
 ## Checklist
 * [ ] Office Hours
-  * [ ] off (default, spell casted after deploying, pause delay 60 sec.)
+  * [ ] Off (default, spell casted after deploying, pause delay 60 sec.)
 * [ ] Exec Hash
   * [ ] empty (default)
 * [ ] 30 Days Expiry
-  * [ ] weekly (monthly discarded)
+  * [ ] weekly
+  * [ ] monthly (discarded)
```

Comment:
Not much context here as to what discarded means.. 

If we don't use monthly, we shouldn't have it as an option
If you mean 'discard' weekly in favour of monthly, we don't need to have it in brackets here

Reply:
yeah, we should probably remove it.

Reply:
done in 1765987, where I removed both `weekly` and `monthly` as now the labels have been removed in `config.sol` (see [old spells](https://github.com/makerdao/spells-mainnet/blob/master/archive/2021-05-14-DssSpell/DssSpell.t.sol#L137) in archive for reference for labels and expiration)